rp2040-hal/src/gpio/pin.rs

```diff
@@ -1075,7 +1075,55 @@ gpio!(
 /// [here](self#types) for a list of the available [`PinMode`] type aliases.
 ///
 /// # Example
-/// ...
+/// Calling the macro like this:
+/// ```rust
+/// bsp_pins! {
+///     #[cfg(feature != "gpio")]
+///     Gpio0 {
+///          /// Doc gpio0
+///          name: gpio0,
+///          aliases: { FunctionPio0: PioPin }
+///     },
+///     Gpio1 {
+///          name: led,
+///          aliases: { FunctionPwm: LedPwm }
+///      },
+/// }
+/// ```
+///
+/// Is roughly equivalent to the following source code (excluding the docs strings below):
+/// ```
+/// use ::rp2040_hal as hal;
+/// use hal::gpio;
+/// pub struct Pins {
+///     # [cfg (feature != "gpio")]
```

Comment:
`#[cfg(not(feature = "gpio"))]` ?

Reply:
Yeah some hacky workaround, I removed it already